src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -303,9 +303,6 @@ void ISimulation<ImplementationType>::run()
     // Determine if we have to use the preprocessors at least one time.
     pData.initialize(study.parameters);
 
-    // Prepro only ?
-    ImplementationType::preproOnly = settings.tsGeneratorsOnly;
-
```

Comment:
Change of behavior here.

Reply:
Thanks.
Maybe I mistook this for local assignation but its parentType::member assignation :/